docs/examples/notebooks/02-set_neuron_params.ipynb

```diff
@@ -0,0 +1,281 @@
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "\n",
+    "import nengo\n",
+    "from nengo.solvers import NoSolver\n",
+    "\n",
+    "import nengo_fpga\n",
+    "from nengo_fpga.networks import FpgaPesEnsembleNetwork"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 2: Choose an FPGA Device\n",
+    "\n",
+    "Define the FPGA device on which the remote FpgaPesEnsembleNetwork will run. This name corresponds with the name in your `fpga_config` file. Recall that in the `fpga_config` file, device names are identified by the square brackets (e.g., **[de1]** or **[pynq]**). The names defined in your configuration file might differ from the example below. Here, the device **de1** is being used."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "board = 'de1'  # Change this to your desired device name"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 3: Create Some Weights\n",
+    "\n",
+    "In order to demonstrate the mechanism by which to set encoder and decoder matrices on the FPGA we will use Nengo to generate some values. We will set the encoders as desired and use the Nengo solver to find the decoders. In practice, you can generate your matrices however you like, though keep in mind you likely want to set `bias`, `encoders`, `gain`, and `decoders` together in order to have computationally functional model.\n",
+    "\n",
+    "In this example we will use 4 neurons with 2 dimensions. Each neuron will represent one quadrant of a sine and cosine period."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Dummy network to generate neuron params\n",
+    "with nengo.Network(seed=1) as param_net:\n",
+    "    # Ensemble with our desired encoders\n",
+    "    a = nengo.Ensemble(\n",
+    "        n_neurons=4,\n",
+    "        dimensions=2,\n",
+    "        neuron_type=nengo.RectifiedLinear(),\n",
+    "        encoders=[[0.707, 0.707], [-0.707, 0.707],\n",
+    "                  [-0.707, -0.707], [0.707, -0.707]],\n",
+    "        intercepts=nengo.dists.Choice([0]),\n",
+    "        max_rates=nengo.dists.Choice([100])\n",
+    "        )\n",
+    "\n",
+    "    # We need to create a connection to solve decoders\n",
+    "    b = nengo.Node(size_in=2)\n",
+    "    conn = nengo.Connection(a, b)\n",
+    "\n",
+    "# Decoders are solved once we create the simulator object\n",
+    "with nengo.Simulator(param_net) as param_sim:\n",
+    "    bias = param_sim.data[a].bias\n",
+    "    encoders = param_sim.data[a].encoders\n",
+    "    gain = param_sim.data[a].gain\n",
+    "    decoders = param_sim.data[conn].weights"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 4: Create the FPGA Network"
```

Comment:
.. FPGA Ensemble 
(just to be consistent with the other notebook examples)

Reply:
Also a little blurb about the code:

The network created in the previous step was used generate the neuron parameters and is not intended to be run on the FPGA. Here, the actual Nengo model that is to be run on the FPGA is created. Note that for this example, the decoders have been initialized to zero in order to demonstrate the effect of using a network with a decoder weight matrix of zero versus a network using the pre-generated decoder weights from the previous step.

Reply:
^ @bmorcos  I dont think you addressed this comment.

Reply:
Oops, not sure how I missed this.